canvas_account_admin_tools/settings/base.py

```diff
@@ -326,7 +330,7 @@
     'loggers': {
         'gunicorn': {
             'handlers': ['gunicorn', 'console'],
-            'level': 'INFO',
+            'level': logging.WARNING,
```

Comment:
I assume this is to reduce the noisiness of the `gunicorn` logs?

Reply:
Yeah -- we send tons of gunicorn log events to Splunk but they're not that useful. I think we really only need to know about warnings and above.